rosbag2_tests/test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp

```diff
@@ -559,7 +559,7 @@ TEST_F(RecordFixture, record_fails_gracefully_if_bag_already_exists) {
 TEST_F(RecordFixture, record_fails_if_both_all_and_topic_list_is_specified) {
   internal::CaptureStderr();
   auto exit_code =
-    execute_and_wait_until_completion("ros2 bag record -a /some_topic", temporary_dir_path_);
+    WEXITSTATUS(std::system("ros2 bag record -a /some_topic"));
```

Comment:
why this change? Same below

Reply:
These tests were failing and that fix gave me a placebo of fixing the root cause, apparently the issue was somewhere else. Double checked local and that is not needed anymore, should be reverted back now.